ad9361/sw/platform_xilinx/platform.c

```diff
@@ -134,7 +134,7 @@ int32_t spi_read(struct spi_device *spi,
 	XSpiPs_PolledTransfer(&spi_instance, data, data, bytes_number);
 #else
 	uint32_t cnt = 0;
-#ifdef XPAR_AXI_SPI_0_DEVICE_ID
+#ifdef XPAR_SPI_0_DEVICE_ID
```

Comment:
maybe do `#if defined(XPAR_AXI_SPI_0_DEVICE_ID) || defined(XPAR_SPI_0_DEVICE_ID)`
that way, this code remains backwards compatible with the old define ( if people want to run it there as well )